### DISCUSSION TITLE
Secman in the Future

### DISCUSSION THREAD

**Discussion by abdfnx** [scmn-dev/secman#34] [Ideas]
# Secman Now

<img src="https://assets.secman.dev/assets/Secman.svg" />

# Secman in the Future

<img src="https://assets.secman.dev/assets/Secman-Desktop.png" />

> I wish this, so let's keep moving...

**Comment by Timothee-Cardoso** [comment]
If we did it, **secman** well be best password manager in the history man.

**Reply by abdfnx** [reply]
Yeah man, I very wish this
